src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -226,7 +232,7 @@ def builtin_neighbors(
         outputs={"__result"},
     )
     idx_name = unique_var_name()
-    sdfg.add_scalar(idx_name, dace.int64, transient=True)
+    sdfg.add_scalar(idx_name, _INDEX_DTYPE, transient=True)
```

Comment:
I am not sure if it is an issue, but here you go down with the precision, since `_TYPE_MAPPING["int"]` is [not necessarily 64bits](https://github.com/edopao/gt4py/blob/52f2abb24f3ee3007234c5638b6484b3076e12fd/src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py#L52C30-L52C70). But it might be irrelevant.